Dockerfile.ci-unified

```diff
@@ -0,0 +1,52 @@
+ARG REGISTRY
```

Comment:
bug: `VERSION` has been dropped - so the RP is no longer being passed a git commit.

Reply:
fix this issue